### DISCUSSION TITLE
STweep removed dangling END_ statement at the end of the file

### DISCUSSION THREAD

**Discussion by Roald87** [STweep/STweep#43] [Q&A]
Not sure if it is intended behavior, but I noticed that when I format code which has a non-matching `END_*` it will remove the `END_*`. It only does this when the `END_*` statement is at the end of the file.

For example 

```
//in
a:=2;
END_FOR

// out
a:=2;
```

Is this intended?

**Comment by Barteling** [comment]
It is not,
I'll check it. STweep has sometimes some problems with code with parsing errors.

**Comment by Roald87** [comment]
It doesn't happen if there is a `;` behind the `END_`.
